var/spack/repos/builtin/packages/xxdiff/package.py

```diff
@@ -0,0 +1,35 @@
+# Copyright 2013-2023 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class Xxdiff(MakefilePackage):
+    """Graphical File And Directories Comparator And Merge Tool."""
+
+    homepage = "https://furius.ca/xxdiff/"
+    git = "https://github.com/blais/xxdiff.git"
+
+    maintainers("vanderwb")
+
+    version("master", branch="master")
+    version("2023-01-10", commit="604300e")
+
+    depends_on("flex@2.5.31:")
+    depends_on("bison")
```

Comment:
Are `flex` and `bison` are often just build dependencies. Are they also link-time dependencies by xxdiff? 
(If you don't specify a type Spack defaults to `("build", "link")`.